aexpect/client.py

```diff
@@ -1130,14 +1130,15 @@ def cmd_output(self, cmd, timeout=60, internal_timeout=None,
         self.sendline(cmd)
         try:
             o = self.read_up_to_prompt(timeout, internal_timeout, print_func)
-        except ExpectError as e:
-            o = self.remove_command_echo(e.output, cmd)
-            if isinstance(e, ExpectTimeoutError):
-                raise ShellTimeoutError(cmd, o)
-            elif isinstance(e, ExpectProcessTerminatedError):
-                raise ShellProcessTerminatedError(cmd, e.status, o)
-            else:
-                raise ShellError(cmd, o)
+        except ExpectTimeoutError as error:
+            output = self.remove_command_echo(error.output, cmd)
+            raise ShellTimeoutError(cmd, output)
+        except ExpectProcessTerminatedError as error:
+            output = self.remove_command_echo(error.output, cmd)
+            raise ShellProcessTerminatedError(cmd, error.status, output)
+        except ExpectError as error:
+            output = self.remove_command_echo(error.output, cmd)
+            raise ShellError(cmd, output)
```

Comment:
Sure, the only small problem is that we have to repeat a bit the 'output = ' line.

Reply:
yes, but by disabling the check we can hide missing attributes (which actually happened to me while addressing this due to minor mistake I did. That's why I chose this way :smile: